### ISSUE TITLE
Speedmenu layout in landscape mode

### ISSUE THREAD

**Issue by defini-tiv** [TeamNewPipe/NewPipe#4913]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

I think the speed control should have another layout  in landscape mode, so most users don't have to scroll.
There is an image how it could look

#### Is your feature request related to a problem? Please describe it

At the moment you have to scroll through the menu and see not all options at the same time. 
I have to admit that my phone screen may be a bit thin and long but i think it is the same issue on many other devices that the menu is too big in vertical and has enough space left on the sites. 

#### Additional context

At the time:
![old_speed-menu](https://user-images.githubusercontent.com/68238058/99536558-d5053700-29aa-11eb-9b75-edd666b00c4b.png)
My suggestion:
![new_speed-menu](https://user-images.githubusercontent.com/68238058/99536574-d9315480-29aa-11eb-998a-1e6ce1168ac0.png)

#### How will you/everyone benefit from this feature?

Its just a detail, but i think it would improve UX because you see everything after opening it up and don't have to scroll anymore - it would just feel cleaner.

**Comment by MD77MD** [comment]
good catch😎

**Comment by SameenAhnaf** [comment]
The controls' menu could be a lot more compact for both landscape and portrait mode. Here is my proposal.
![IMG_20210509_052306](https://user-images.githubusercontent.com/79245641/117556072-d3c5e480-b086-11eb-98d4-5def37e8ddcd.jpg)

Sorry for poor editing.

Key Changes:
1. Interchanging among 6 steps (like Fit/Fill/Zoom) is a huge hassle. That's why, I suggested two types of steps.
A. Small step: Interchange among 1%, 5% and 10%
B. Big step: Interchange among 25%, 50% and 100%
The selected step could be shown in bold.
2. Single tap on 'Step' will reset the speed and pitch.
3. Selecting a step should show the possible values with that step. Just like step, last selected speed should be shown in bold. Scroll up or down for more speed options. A toast should appear while changing the pitch as it is normally intended for changing speed.
A. Single tap: Change only the speed.
B. Double tap: Change both speed and pitch accordingly.
C. Long press: Change only the pitch.
4. "Fast forward during silence" could be replaced by "FF during silence".
5. 'Unhook' should always stay enabled for single tap and long press gesture. There's already "Double tap" gesture as a replacement.

Pros:
1. Same playback controls on any mode (landscape and portrait) and any player (main, popup and queue manager).
2. Much more compact.
3. Similar with most video players (except the checkboxes)

Cons:
1. Speed and pitch need to be changed twice if separate values are required. Normally, drop-down menu should get closed right after choosing any option. Let me know your thoughts.

**Comment by litetex** [comment]
After reviewing #7351 I want to note a few things that should be considered:
* Having 2 layouts causes 2x maintenance effort.<br/>There have been similar problems in the past when something was added to ``player``: Someone forgot to add a button on the large-player and the whole app crashed on TV and tablets.
* The dialog is adjusted to the device. Some devices will always have this problem particularly small/old devices.
* With different languages the required text-area sometime gets larger. Which then also causes problems.

So I would [keep it stupid simple](https://en.wikipedia.org/wiki/KISS_principle):
Keep one layout and remove some useless spaces/margins from that layout.

I did that with a few simple changes and I think the results are actually pretty good:
![grafik](https://user-images.githubusercontent.com/40789489/140398225-2257b793-ffa9-4891-8d0c-8c62ff3cb243.png)
